web/src/user_profile.ts

```diff
@@ -548,6 +551,82 @@ export function show_user_profile_access_error_modal(): void {
     });
 }
 
+function add_user_to_groups(group_ids: number[], user_id: number, $alert_box: JQuery): void {
+    const group_ids_successfully_added: number[] = [];
+
+    function add_user_to_next_group(): void {
+        if (group_ids_successfully_added.length >= group_ids.length) {
+            if (group_ids_successfully_added.length > 0) {
+                ui_report.success(
+                    $t_html({
+                        defaultMessage: "Successfully added to group(s)!",
```

Comment:
"Successfully updated membership" might be a bit nicer for translators.

Reply:
This is better, since we're looking at a user, not a group:
``` diff
diff --git a/web/src/user_profile.ts b/web/src/user_profile.ts
index e2b7277d2b..24b7440af8 100644
--- a/web/src/user_profile.ts
+++ b/web/src/user_profile.ts
@@ -559,7 +559,7 @@ function add_user_to_groups(group_ids: number[], user_id: number, $alert_box: JQ
             if (group_ids_successfully_added.length > 0) {
                 ui_report.success(
                     $t_html({
-                        defaultMessage: "Successfully added to group(s)!",
+                        defaultMessage: "Added successfully!",
                     }),
                     $alert_box,
                     1200,
```